lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -77,7 +82,8 @@ def _make_mask_cube(
                                                long_name=coord_name,
                                                units=Unit(topographic_units))
         mask_cube.add_aux_coord(threshold_coord)
-    mask_cube.attributes['Topographical Type'] = key.title()
+    mask_cube.attributes.update(
+        {'topographic_zones_include_seapoints': str(sea_points_included)})
```

Comment:
Maybe comment here as you did on the PR - string conversion for iris' benefit.

Reply:
Added